apps/electron-app/app-update.yml

```
@@ -0,0 +1,3 @@
+provider: github
+owner: co-browser
+repo: vibe
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)